### PR TITLE
Prepare release

### LINKED ISSUE #5: cannot install the collection
##### SUMMARY
Seems like sops is not added to galaxy:

```
ansible-galaxy collection install community.sops
Process install dependency map
ERROR! Failed to find collection community.sops:*
```

##### ISSUE TYPE
- Installation issue

##### COMPONENT NAME
community.sops

##### ANSIBLE VERSION

```

ansible --version
ansible 2.9.10
  config file = None
  python version = 3.7.6 (default, Dec 30 2019, 19:38:26) [Clang 11.0.0 (clang-1100.0.33.16)]

```

##### OS / ENVIRONMENT
macOS

##### STEPS TO REPRODUCE
```
ansible-galaxy collection install community.sops
```

### PR DESCRIPTION
**Pull request by endorama**
### Motivation

Release to allow installing the collection using Ansible Galaxy. 

### Changes description

- improve `README`
- add more tags in `galaxy.yml`

## Additional notes

Closes #5

**Note**: this PR will not be merged until the `sops` `vars` plugin has been merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,22 +2,32 @@
...
 
+- `devel`
+
 ## External requirements
 
 <!-- List any external resources the collection depends on, for example minimum versions of an OS, libraries, or utilities. Do not list other Ansible collections here. -->
 
-### Supported connections
-<!-- Optional. If your collection supports only specific connection types (such as HTTPAPI, netconf, or others), list them here. -->
+You will need to install [`sops`](https://github.com/mozilla/sops) manually before using plugins provided by this
```
Comment (gundalow) [inline README.md:19]: Is there a link to that Sops installation page?
Is there a minimum (or maximum) version of Sops that this collection requires?

File: README.md [hunk 2]
```diff
@@ -2,22 +2,32 @@
...
+The `community.sops` collection allows integrating [`mozilla/sops`](https://github.com/mozilla/sops) in Ansible.
 
+`mozilla/sops` is a tool for encryption and decryption of files using secure keys (GPG, KMS). It can be leveraged in Ansible to provide an easy to use and flexible to manage way to manage ecrypted secrets' files.
+ 
 ## Tested with Ansible
 
 <!-- List the versions of Ansible the collection has been tested with. Must match what is in galaxy.yml. -->
 
+- `devel`
+
```
Comment (gundalow) [inline README.md:14]: Can we please add 2.10 here and into CI testing?

**Comment by felixfontein** [comment]
BTW, I wrote up the basic steps for releasing a collection here: https://github.com/ansible/community/wiki/ReleasingCollections#releasing-without-release-branches-for-smaller-collections

File: README.md (hunk 2 above)
Comment (felixfontein) [reply README.md:14]: Should be simpe to do by adding a matrix variable which has values `devel` and `stable-2.10` (and maybe also `stable-2.9`). It should be used in lines https://github.com/ansible-collections/community.sops/blob/main/.github/workflows/ansible-test.yml#L21 and https://github.com/ansible-collections/community.sops/blob/main/.github/workflows/ansible-test.yml#L74

Comment (gundalow) [reply README.md:14]: Can copy from https://github.com/ansible-collections/collection_template/blob/main/.github/workflows/ansible-test.yml

File: README.md (hunk 1 above)
Comment (endorama) [reply README.md:19]: This is a very interesting question. I would love to have a test suite where multiple `sops` versions are tested, but I'm still struggling with the tests :) 

But as the CLI interface is not expected to change, I would expect any `3.x` `sops` release to be working. I'll add it to the README.

Comment (felixfontein) [reply README.md:19]: For the initial 0.1.0 release, it's totally fine if not everything is perfect :)

File: .github/workflows/ansible-test.yml [hunk 3]
```diff
@@ -13,15 +28,19 @@ jobs:
...
-        run: pip install https://github.com/ansible/ansible/archive/devel.tar.gz --disable-pip-version-check
+      - name: Install ansible-base (${{ matrix.ansible }})
+        run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
 
+      # run ansible-test sanity inside of Docker.
+      # The docker container has all the pinned dependencies that are required.
+      # Explicity specify the version of Python we want to test
       - name: Run sanity tests
-        run: ansible-test sanity --docker -v --color --python 3.6
+        run: ansible-test sanity --docker -v --color --python ${{ matrix.python }}
```
Comment (felixfontein) [inline .github/workflows/ansible-test.yml:42]: The `--python` option should not be used for sanity tests when `--docker` is there. (And you can fix the Python version to 3.8 in the Python setup above.)
```suggestion
        run: ansible-test sanity --docker -v --color
```

File: .github/workflows/ansible-test.yml [hunk 4]
```diff
@@ -53,6 +71,18 @@ jobs:
...
+          # Testing against `devel` may fail as new tests are added.
+          - stable-2.9
+          - stable-2.10
+          - devel
+        python:
+          - 2.7
+          - 3.7
+          - 3.8
+        exclude:
+          - python: 3.8  # blocked by ansible/ansible#70155
```
Comment (felixfontein) [inline .github/workflows/ansible-test.yml:85]: That's not needed anywhere. 70155 only blocks `ansible-test sanity` when run without `--docker` in Python 3.8.

**Comment by endorama** [comment]
So much green in here :D Are all these tests worth it? Should I remove `ubuntu1604` from them?

**Comment by felixfontein** [comment]
You are running most tests multiple times. For sanity and integration, you can drop all Python variants, since they are **not** used in the tests themselves.

File: .github/workflows/ansible-test.yml [hunk 5]
```diff
@@ -4,24 +4,41 @@ on:
...
+        ansible:
+          # It's important that Sanity is tested against all stable-X.Y branches
+          # Testing against `devel` may fail as new tests are added.
+          - stable-2.9
+          - stable-2.10
+          - devel
+        python:
+          - 2.7
+          - 3.7
+          - 3.8
```
Comment (felixfontein) [inline .github/workflows/ansible-test.yml:19]:
```suggestion
```

File: .github/workflows/ansible-test.yml [hunk 6]
```diff
@@ -53,6 +70,16 @@ jobs:
...
```
Comment (felixfontein) [inline .github/workflows/ansible-test.yml:82]:
```suggestion
```

**Comment by endorama** [comment]
@felixfontein my bad, I misunderstood your previous comment! 

Pipeline is failing on `ansible-base` installation because `ansible` is already installed, is this expected?

File: .github/workflows/ansible-test.yml [hunk 7]
```diff
@@ -4,24 +4,35 @@ on:
...
+        run: sudo apt-get remove ansible -y
 
-      - name: Install ansible-base (devel)
-        run: pip install https://github.com/ansible/ansible/archive/devel.tar.gz --disable-pip-version-check
+      - name: Install ansible-base (${{ matrix.ansible }})
+        run: sudo pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
 
+      # run ansible-test sanity inside of Docker.
+      # The docker container has all the pinned dependencies that are required.
+      # Explicity specify the version of Python we want to test
```
Comment (felixfontein) [inline .github/workflows/ansible-test.yml:32]:
```suggestion
```

File: .github/workflows/ansible-test.yml [hunk 8]
```diff
@@ -70,14 +87,19 @@ jobs:
...
+
+      - name: Install ansible-base (${{ matrix.ansible }})
+        run: sudo pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
 
       - name: Run integration tests in ${{ matrix.docker_container }} Docker container
         run: ansible-test integration --docker ${{ matrix.docker_container }} -v --color --retry-on-error --continue-on-error --diff --coverage
+        working-directory: ./ansible_collections/community/sops
 
       - name: Generate coverage report.
         run: ansible-test coverage xml -v --requirements --group-by command --group-by version
```
Comment (felixfontein) [inline .github/workflows/ansible-test.yml:101]: @gundalow for ansible-base >= 2.10, another step is needed: https://github.com/ansible-collections/community.general/blob/main/tests/utils/shippable/shippable.sh#L153

Not that urgent though :)

File: README.md [hunk 9]
```diff
@@ -2,22 +2,32 @@
...
 <!-- Describe the collection and why a user would want to use it. What does the collection do? -->
+The `community.sops` collection allows integrating [`mozilla/sops`](https://github.com/mozilla/sops) in Ansible.
 
+`mozilla/sops` is a tool for encryption and decryption of files using secure keys (GPG, KMS). It can be leveraged in Ansible to provide an easy to use and flexible to manage way to manage ecrypted secrets' files.
+ 
 ## Tested with Ansible
 
 <!-- List the versions of Ansible the collection has been tested with. Must match what is in galaxy.yml. -->
 
+- `devel`
```
Comment (felixfontein) [inline README.md:13]:
```suggestion
- `devel`
- latest 2.9 release
- latest 2.10 release
```

**Comment by felixfontein** [comment]
OK, CI is green now! (I'm assuming that ansible/third-party-check won't let me down ;) )

How about we merge this now, and re-run tests for #6? Not that much can fail there yet (it has no tests yet) though, but still...

**Review by felixfontein — APPROVED** [review]

**Pull request merged**